src/Engine/Util/Util.cpp

```diff
@@ -0,0 +1,3 @@
+#include "Util.hpp"
+
+constexpr float Util::infinity;
```

Comment:
@Daskie correct me if I'm wrong, Util is deprecated?

Reply:
Util.cpp doesn't even exist on master. This should be gone

Reply:
If we will keep Util.hpp: Util.hpp is in master. The program will not compile if we do not have Util.cpp and these lines

Reply:
I looked, some things use Util::infinity, so it looks like we need to keep Util.hpp in. If Util.hpp is kept in, we will need to keep Util.cpp in as well, or else the program will not compile on OSX or Linux

Reply:
This kind of defeats the purpose of having it as a variable. Can you delete Util.cpp, and add change the infinity constant out for an infinity() function?